### PR TITLE
Frontend/fix transfer time

### LINKED ISSUE #1684: Share experience via twitter after bridge transfer
## Description
As a User, I want to share my bridging experience via Twitter once I completed a bridge transfer to share the positive user experience and speed of Beamer Bridge.

## Acceptance criteria
The information displayed on the toast is also sharable via twitter. #1209
Users can share this experience with a prewritten text on Twitter (i have seen this on some other pages, so this might be a Twitter feature.) that includes the information mentioned above (don't include fees or amount transferred)

### PR DESCRIPTION
**Pull request by GabrielBuragev**
Changed the time strategy: from using `block.timestamp` to using local clock (with `Date.now()`).
You can read more on why this was necessary here: https://github.com/beamer-bridge/beamer/issues/1796

Closes #1684 & #1796

### CODE REVIEW

File: frontend/src/actions/transfers/transfer.ts [hunk 1]
```diff
@@ -343,10 +347,8 @@ export class Transfer extends MultiStepAction implements Encodable<TransferData>
     );
 
     try {
-      const timestamp = await Promise.race([fulfillmentPromise, expirationPromise]);
-      if (timestamp) {
-        this._requestFulfillment = new RequestFulfillment({ timestamp });
-      }
+      await Promise.race([fulfillmentPromise, expirationPromise]);
+      this._requestFulfillment = new RequestFulfillment({ timestamp: Date.now() });
```
Comment (manuelwedler) [inline frontend/src/actions/transfers/transfer.ts:351]: Could you chain the `new Request Fulfillment` call to the `fulfillmentPromise` with a `then`? So we only set the fulfillment if the request was really fulfilled. I think this is the safer approach for future changes.

**Review by manuelwedler — CHANGES_REQUESTED** [review]
Nice fix, thank you! Just one little change I would like to see getting addressed.

File: frontend/src/actions/transfers/transfer.ts (hunk 1 above)
Comment (GabrielBuragev) [reply frontend/src/actions/transfers/transfer.ts]: Good catch! ✅

**Review by compojoom — APPROVED** [review]
lgtm

**Review by manuelwedler — APPROVED** [review]
Looks like I have to approve another time since I requested the changes.

**Pull request merged**